src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorStatementOfTruthPaySubmit.java

```diff
@@ -110,6 +134,20 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .build();
     }
 
+    public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
+                                               CaseDetails<CaseData, State> beforeDetails) {
+        final CaseData caseData = details.getData();
+        final PaymentStatus paymentStatus = caseData.getPayments().get(caseData.getPayments().size() - 1).getValue().getPaymentStatus();
```

Comment:
Feels like we might need something a bit more robust here? Sum the total of successful payments and make sure it is the same value as the Fee?